compiler/crates/relay-typegen/tests/generate_typescript/fixtures/unmasked-fragment-spreads.expected

```diff
@@ -0,0 +1,87 @@
+==================================== INPUT ====================================
+fragment UserProfile on User {
+  profilePicture(size: $ProfilePicture_SIZE) {
+    ...PhotoFragment @relay(mask: false)
+
+    # duplicated field should be merged
+    ...AnotherRecursiveFragment @relay(mask: false)
+
+    # Compose child fragment
+    ...PhotoFragment
+  }
+}
+
+fragment PhotoFragment on Image {
+  uri
+  ...RecursiveFragment @relay(mask: false)
+}
+
+fragment RecursiveFragment on Image @relay(mask: false) {
+  uri
+  width
+}
+
+fragment AnotherRecursiveFragment on Image {
+  uri
+  height
+}
+==================================== OUTPUT ===================================
+import { FragmentReference } from "relay-runtime";
+export type AnotherRecursiveFragment$ref = FragmentReference & { _: "AnotherRecursiveFragment$ref" };
+export type AnotherRecursiveFragment$fragmentType = AnotherRecursiveFragment$ref & { _: "AnotherRecursiveFragment$fragmentType" };
+export type AnotherRecursiveFragment = {
+  readonly uri?: string,
+  readonly height?: number,
+  readonly $refType: AnotherRecursiveFragment$ref,
+};
+export type AnotherRecursiveFragment$data = AnotherRecursiveFragment;
+export type AnotherRecursiveFragment$key = {
+  readonly $data?: AnotherRecursiveFragment$data,
+  readonly $fragmentRefs: AnotherRecursiveFragment$ref,
+};
+-------------------------------------------------------------------------------
+import { FragmentReference } from "relay-runtime";
+export type PhotoFragment$ref = FragmentReference & { _: "PhotoFragment$ref" };
+export type PhotoFragment$fragmentType = PhotoFragment$ref & { _: "PhotoFragment$fragmentType" };
+export type PhotoFragment = {
+  readonly uri?: string,
+  readonly width?: number,
+  readonly $refType: PhotoFragment$ref,
+};
+export type PhotoFragment$data = PhotoFragment;
+export type PhotoFragment$key = {
+  readonly $data?: PhotoFragment$data,
+  readonly $fragmentRefs: PhotoFragment$ref,
+};
+-------------------------------------------------------------------------------
+import { FragmentReference } from "relay-runtime";
+export type RecursiveFragment$ref = FragmentReference & { _: "RecursiveFragment$ref" };
+export type RecursiveFragment$fragmentType = RecursiveFragment$ref & { _: "RecursiveFragment$fragmentType" };
+export type RecursiveFragment = {
+  readonly uri?: string,
+  readonly width?: number,
+};
+export type RecursiveFragment$data = RecursiveFragment;
+export type RecursiveFragment$key = {
+  readonly $data?: RecursiveFragment$data,
+  readonly $fragmentRefs: RecursiveFragment$ref,
+};
+-------------------------------------------------------------------------------
+import type { PhotoFragment$ref } from "PhotoFragment.graphql";
```

Comment:
It would be better to avoid any need to import these types, as that entirely removes the need to have a single artefact directory (or Haste). See for example [this output](https://github.com/relay-tools/relay-compiler-language-typescript/blob/e1b092f081d4e816ecd17fb0d3b4e62cdf3db917/test/__snapshots__/TypeScriptGenerator-test.ts.snap#L495) of the existing TS plugin.

Reply:
Yeah... I think this is actually a blocker for a merge. Typescript wouldn't know what that means (please correct me here).

At the very least just prefix with a `./`, 

```diff
-import type { PhotoFragment$ref } from "PhotoFragment.graphql";
+import type { PhotoFragment$ref } from "./PhotoFragment.graphql";
```

Reply:
Hmm, yeah that’s a good point. @kassens can you amend that in your import?